code/modules/admin/verbs/adminjump.dm

```diff
@@ -46,14 +79,11 @@
 	if(!isobserver(usr))
 		message_admins("[key_name_admin(usr)] jumped to [T.x], [T.y], [T.z] in [T.loc]", 1)
 	admin_forcemove(usr, T)
-	feedback_add_details("admin_verb","JT") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
-	return
+	feedback_add_details("admin_verb", "JT") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-/client/proc/jumptomob(var/mob/M in GLOB.mob_list)
-	set category = "Admin"
+/client/proc/jumptomob(mob/M)
 	set name = "Jump to Mob"
```

Comment:
```suggestion
```
Since it's hidden now

Reply:
I tried this, but the proc also shows up in the right click menu for mobs, so it still needs to have a name set.